modules/programs/zsh.nix

```diff
@@ -728,21 +728,21 @@ in
           ${if cfg.autocd != null then "${if cfg.autocd then "setopt" else "unsetopt"} autocd" else ""}
         ''))
 
-        (lib.mkIf (cfg.initExtra != "") (mkOrder 1200 cfg.initExtra))
+        (lib.mkIf (cfg.initExtra != "") cfg.initExtra)
```

Comment:
Removing custom prio here allows migration to be easier for `initExtra` to `initContent`